.github/workflows/publish-v2.yml

```diff
@@ -78,8 +78,8 @@ jobs:
         run: npm run install:all
 
       #Build and package
-      - name: Build 
-        run: npm run webpack
+      - name: Package 
```

Comment:
💡thoughts / idea: i wonder if you keep both steps, the webpack is normal build step and then package step?